package.json

```diff
@@ -16,6 +16,7 @@
     "build-storybook": "STORYBOOK=true storybook build"
   },
   "dependencies": {
+    "@csstools/postcss-global-data": "^3.0.0",
```

Comment:
I think this should be in `devDependencies`

Reply:
Fixed in later commit